### DISCUSSION TITLE
Add other Organization members to the current organization

### DISCUSSION THREAD

**Discussion by gokulraghuthaman** [zitadel/zitadel#7069] [Q&A] [answered]
Hi,
I am trying to add another organization member to current organization with roles using API.
I used https://$CUSTOM-DOMAIN/admin/v1/members api for this as an admin, but the added member is not showing in the manager list or in users list.
How to add other organization members to the current organization properly using the api? and to get the users inside the default organization which api we can use?

**Comment by hifabienne** [comment, marked as answer]
Hei @gokulraghuthaman 
There are different levels of members. Instance, Organization and project. 
At the moment we do have separat apis depending on what you want to manage.
So in your case you need this request: https://zitadel.com/docs/apis/resources/mgmt/management-service-add-org-member
In the future we will rework the apis, so you have them ressource based, an no matter if you manage the instance or an organization you can use the same api.

**Reply by gokulraghuthaman** [reply]
Thank you so much for your help! Your answer was really helpful and clarified things for me. I can add the users in another organisation to the current one as the manager, but from the instance level as the manager of the team with permission. I can add the user in the user list only by adding new user, not by user from other organisation.
